src/librustc_mir/transform/erase_regions.rs

```diff
@@ -143,9 +143,8 @@ impl<'a, 'tcx> EraseRegions<'a, 'tcx> {
             Rvalue::Use(ref mut operand) => {
                 self.erase_regions_operand(operand)
             }
-            Rvalue::Repeat(ref mut operand, ref mut constant) => {
+            Rvalue::Repeat(ref mut operand, _) => {
                 self.erase_regions_operand(operand);
```

Comment:
Since a `Ty` is stored in `TypedConstVal`, you’ll have to erase regions in that. Erasing them inline (similarly to what’s done in `SwitchTy` case above) is fine, I think.